armi/reactor/converters/blockConverters.py

```diff
@@ -216,6 +216,19 @@ class ComponentMerger(BlockConverter):
     .. impl:: Homogenize one component into another.
         :id: I_ARMI_BLOCKCONV0
         :implements: R_ARMI_BLOCKCONV
+
+        This subclass of ``BlockConverter`` is meant as a one-time-use tool, to convert
+        a multi-component ``Block`` into one ``Component``. This means averaging the
```

Comment:
A little bit confused by "a multi-component Block" mentioned in ComponentMerger and "a multiple Components" in the description for MultipleComponentMerger. 

Does the ComponentMerger only merge single solute component into the solvent component?

Reply:
I'll try to clean up the language. 1 sec.